baystation12.dme

```diff
@@ -817,7 +817,6 @@
 #include "code\game\objects\auras\personal_shields\personal_shield.dm"
 #include "code\game\objects\effects\aliens.dm"
 #include "code\game\objects\effects\bump_teleporter.dm"
-#include "code\game\objects\effects\corruption.dm"

```

Comment:
this was an obsolete file containing some temporary objects ketrai first made to test corruption sprites